Detections/AuditLogs/PrivlegedRoleAssignedOutsidePIM.yaml

```diff
@@ -21,7 +21,8 @@ tags:
 query: |
   AuditLogs
   | where Category =~ "RoleManagement"
-  | where ActivityDisplayName =~ 'Add member to role (permanent)'
+  | where OperationName startswith "Add member to role outside of PIM"
```

Comment:
This seems like a good idea, but let's use `has` here instead of `startswith`.

Reply:
Done.

Reply:
@ep3p - great thanks.  I will approve.